contributing_to_docs/term_glossary.adoc

```diff
@@ -143,7 +143,7 @@ old replication controller with this new one.
 Usage: Dockerfile; wrapped with [filename] markup. See
 link:doc_guidelines.html[Documentation Guidelines] for markup information.
 
-Docker can build images automatically by reading the instructions from a
+Container engines (Like: Podman, Buildah, Docker) can build images automatically by reading the instructions from a
```

Comment:
s/(Like: Podman, Buildah, Docker)/, like Podman, Buildah, and Docker,

Our style guide says to avoid () wherever possible, and I think this is a fair substitution.

Reply:
Fixed